### DISCUSSION TITLE
When will Navi24 be supported?

### DISCUSSION THREAD

**Discussion by qingquan119** [ChefKissInc/NootRX#32] [Q&A] [answered]
Driver support from Navid 24 is urgently needed

**Comment by VisualEhrmanntraut** [comment, marked as answer]
Urgently needed? Pay up then; 50k€ to start rushing the development.
I am the only dev, and I have my own life, needs and responsibilities. But I don't have any Navi 24 testers regardless. If you want this to happen, step 0 would be to reach me on Telegram.

**Reply by SiddharthDevare10** [reply]
I have an 6500XT with hackintosh setup without the graphics support up for navi 24 testing

**Reply by VisualEhrmanntraut** [reply]
Ok... message me then?

**Reply by SiddharthDevare10** [reply]
Where should i contact you

**Reply by VisualEhrmanntraut** [reply]
<img width="762" alt="Screenshot 2024-03-13 at 17 02 04" src="https://github.com/ChefKissInc/NootRX/assets/30368284/5a1a26bc-c074-41f9-a4e7-2f3c7ac3beef">

**Reply by OnderPolatGumus** [reply]
ill help too i wrote you on telegram
